feature-libs/customer-ticketing/root/events/customer-ticketing-event.listener.ts

```diff
@@ -45,6 +49,56 @@ export class CustomerTicketingEventListener implements OnDestroy {
     );
   }
 
+  protected onGetTicketCatQueryReset(): void {
```

Comment:
1- for all these created functions, they needed to be called in the constructor

2 - we don't need all these functions, you can place this in one function that has the merges of the stream of `LogoutEvent` and `LoginEvent`, which will fire those events all in one because at this point, it is just DRY

-----
The 1- would make sense if we're making multiple event listeners for each of the `facade`, but it looks like the direction we want to go is to have one facade that encapsulates the whole functionality. I don't have much preference. It is entirely up to you guys on whether you want to have decouple the facade into separate ones to keep the service logic different. If not, then let's go with 2.